smart-contract-example/README.md

```diff
@@ -0,0 +1,90 @@
+- `scripts/` contains a script to deploy a token contract to our local Ethereum network
+- `test/` contains a simple test for our token contract
+- `hardhat.config.ts` configures our Hardhat setup. 
+It allows us to configure Hardhat to use a local Ethereum network created by the `eth-network-package`.
+
+### Running the Example
+
+This assumes you have the following services installed:
+- [Kurtosis CLI](https://docs.kurtosis.com/cli/)
+- [`npx`](https://www.npmjs.com/package/npx)
+- Hardhat
+
+1. Run 
+    ```
+    kurtosis run github.com/kurtosis-tech/eth-network-package
+    ```
+    The output should look something like this
+    ```
+    ========================================== User Services ==========================================
+    UUID           Name                                           Ports                                         Status
+    bbccb71bf6db   cl-client-0-beacon                             http: 4000/tcp -> http://127.0.0.1:64256      RUNNING
+                                                                  metrics: 5054/tcp -> http://127.0.0.1:64257   
+                                                                  tcp-discovery: 9000/tcp -> 127.0.0.1:64258    
+                                                                  udp-discovery: 9000/udp -> 127.0.0.1:59023    
+    a4429f19d246   cl-client-0-validator                          http: 5042/tcp -> 127.0.0.1:64262             RUNNING
+                                                                  metrics: 5064/tcp -> http://127.0.0.1:64263   
+    77c853ab371b   el-client-0                                    engine-rpc: 8551/tcp -> 127.0.0.1:64246       RUNNING
+                                                                  rpc: 8545/tcp -> 127.0.0.1:64248              
+                                                                  tcp-discovery: 30303/tcp -> 127.0.0.1:64247   
+                                                                  udp-discovery: 30303/udp -> 127.0.0.1:58705   
+                                                                  ws: 8546/tcp -> 127.0.0.1:64249               
+    0d8469f528c9   prelaunch-data-generator-1680298734419927594   <none>                                        STOPPED
+    8828574730f6   prelaunch-data-generator-1680298734428736261   <none>                                        STOPPED
+    a9ad4e4cc65c   prelaunch-data-generator-1680298734439099469   <none>                                        STOPPED
+    ```
+    We see a single node with a geth EL client and lighthouse CL client running has been created. The network configuration is configurable via a json file. Read [here](https://github.com/kurtosis-tech/eth-network-package#configuring-the-network) to learn more.
+
+2. Replace `<PORT>` in `hardhat.config.ts` with the port of the rpc uri output from any `el-client-` service. In this case, the port would be `64248`.
```

Comment:
What are the different types of `el-client-` that a user would see?

Reply:
The `eth-network-package` labels each el client service with the name`el-client-<num>` so eg. `el-client-0`, `el-client-1`, `el-client-2`.... In this example since we only spin one node, we see `el-client-0`. Is there a way I can make this more clear? Maybe `el-client-<num>` instead of `el-client-`?

Reply:
Cool that makes sense. `el-client-<num>` and `cl-client-<num>` makes more sense, with `<num>` equal to the instance of that client (whether its EL or CL)